custom-fields/404-page.json

```diff
@@ -0,0 +1,59 @@
+[
+    {
+        "key": "group_5cf45950cc09e",
+        "title": "404 Page",
+        "fields": [
+            {
+                "key": "field_5cf45958bfe40",
+                "label": "404 Page Content",
+                "name": "404_page_content",
+                "type": "wysiwyg",
+                "instructions": "",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "default_value": "<h1 style=\"text-align: center;\"><strong>404<\/strong><\/h1>\r\n<h2 style=\"text-align: center;\">Page Not Found<\/h2>\r\n<h3 style=\"text-align: center;\">We can't find the page you're looking for. Go back or head over the the homepage.<\/h3>",
```

Comment:
There's a typo, it should be "to the homepage"

Reply:
LOL my bad 😅 . I'll fix it